mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -33,21 +40,28 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 	FIELD f_kelwihli knockbackVelocity F
 	FIELD f_kfiskcbr preferredHand Lnet/minecraft/unmapped/C_laxmzoqs;
 	FIELD f_kimgjcqe scoreAmount I
+	FIELD f_kjdgaswv LOGGER Lorg/slf4j/Logger;
 	FIELD f_kkplgjng lastBlockPos Lnet/minecraft/unmapped/C_hynzadkk;
 	FIELD f_kpdzsffg lastHandSwingProgress F
 	FIELD f_krdixdtc serverX D
 	FIELD f_kvmdzdkx prevStepBobbingAmount F
 	FIELD f_lemmcmcu STUCK_ARROW_COUNT Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_lrzhzlmv lastAttackTime I
 	FIELD f_mawomqmy LIVING_FLAGS Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_mcoacjpn SWING_DURATION I
 	FIELD f_mcoihohf itemUseTimeLeft I
 	FIELD f_mfltnpor effectsChanged Z
 	FIELD f_mlsykvgi absorptionAmount F
+	FIELD f_naybbqec ITEM_USE_INTERVAL I
+	FIELD f_noitvgvb BIG_HAT_EXTRA_RENDER_CULLING_SIZE F
 	FIELD f_nzjhzxzv prevLookDirection F
 	FIELD f_obgdercz EQUIPMENT_SLOT_OFFSET I
 	FIELD f_oigploen activeItemStack Lnet/minecraft/unmapped/C_sddaxwyk;
 	FIELD f_okpxkkwu serverHeadYaw D
+	FIELD f_olsxmkqm PLAYER_HURT_EXPERIENCE_TIME I
+	FIELD f_oqcyctiq DEATH_DURATION I
 	FIELD f_ovdhpumu limbDistance F
+	FIELD f_oyncoeen MIN_MOVEMENT_DISTANCE D
```

Comment:
This is `DEFAULT_BASE_GRAVITY` in mojmap

Reply:
Uhm... No its not? It's what I've written